hack/make/release-deb

```diff
@@ -35,6 +35,51 @@ if [[ ! -f "$APTDIR/conf/distributions" ]]; then
 	done > "$APTDIR/conf/distributions"
 fi
 
+# create/update distributions file
+if [[ ! -f "$APTDIR/conf/apt-ftparchive.conf" ]]; then
+	cat <<-EOF > "$APTDIR/conf/apt-ftparchive.conf"
+	Dir {
+		ArchiveDir "${APTDIR}";
+		CacheDir "${APTDIR}/db";
+	};
+
+	Default {
+		Packages::Compress ". gzip bzip2";
+		Sources::Compress ". gzip bzip2";
+		Contents::Compress ". gzip bzip2";
+	};
+
+	TreeDefault {
+		BinCacheDB "packages-\$(SECTION)-\$(ARCH).db";
+		Directory "pool/\$(SECTION)";
+		Packages "\$(DIST)/\$(SECTION)/binary-\$(ARCH)/Packages";
+		SrcDirectory "pool/\$(SECTION)";
+		Sources "\$(DIST)/\$(SECTION)/source/Sources";
+		Contents "\$(DIST)/\$(SECTION)/Contents-\$(ARCH)";
+		FileList "$APTDIR/\$(DIST)/\$(SECTION)/filelist";
+	};
+	EOF
+
+	for suite in $(exec contrib/reprepro/suites.sh); do
+		cat <<-EOF
+		Tree "dists/${suite}" {
+			Sections "main testing experimental";
+			Architectures "i386 amd64";
```

Comment:
Do you think we should put these fields into an array at the top of the file since they're used multiple places?  Then we could use `for arch in "${arches[@]}"; do ...` for iterating and `${arches[*]}` for printing space-separated.